### PR TITLE
Add support for Windows Server core1909

### PR DESCRIPTION
**Pull request by xmapst**
Also added 1909 support for golang:1.14.2-windowsservercore-1909

### CODE REVIEW

**Comment by neolit123** [comment]
@xmapst 
you need to sign the CLA as the bot instructed above, but also please author the commit with your github account / email:
https://github.com/kubernetes-sigs/sig-windows-tools/pull/70/commits/49f27908e78714636a8c4600b66471928ed65e35

**Comment by neolit123** [comment]
/assign @benmoss 
cc @marosset 

i think we should enumerate what platforms are supported in the docs.
https://github.com/kubernetes/website/tree/master/content/en/docs/tasks/administer-cluster/kubeadm
because the users are confused.

i guess adding new platforms means updates in the k/website windows guide too:
>  kubeadm/flannel/hns.psm1 → ...m/flannel/windowsservercore-1809/hns.psm1
> kubeadm/flannel/Dockerfile → ...flannel/windowsservercore-1809/Dockerfile

**Comment by neolit123** [comment]
@xmapst 
one problem with this PR is that it would not automatically build images for kube-proxy, see:
https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/.github/workflows/kube-proxy.yml

it only supports 2019, because github only (?) give us such VMs.

File: kubeadm/golang/windowsservercore-1909/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,73 @@
+FROM mcr.microsoft.com/windows/servercore:1909
```
Comment (benmoss) [inline kubeadm/golang/windowsservercore-1909/Dockerfile:1]: This is fine for now but it'd be cool if we could upstream this to https://github.com/docker-library/golang

**Comment by benmoss** [comment]
Yeah, the big problem with this is sourcing a build server. GitHub actions doesn't have 1909, Appveyor doesn't either.

File: kubeadm/golang/windowsservercore-1909/Dockerfile (hunk 1 above)
Comment (xmapst) [reply kubeadm/golang/windowsservercore-1909/Dockerfile:1]: Ok i will submit a copy to

File: kubeadm/flannel/windowsservercore-1909/hns.psm1 [hunk 2]
```diff
@@ -0,0 +1,502 @@
+#########################################################################
```
[the remark this answers is not in the document]
Comment (xmapst) [reply kubeadm/flannel/windowsservercore-1909/hns.psm1:1]: This uses 1909, slightly different

File: kubeadm/flannel/windowsservercore-1909/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,30 @@
+ARG servercoreTag="1909"
+ARG cniVersion="0.8.5"
+ARG golangTag=1.14.2-windowsservercore-1909
```
[the remark this answers is not in the document]
Comment (xmapst) [reply kubeadm/flannel/windowsservercore-1909/Dockerfile:3]: This is based on the original Dockerfile modification

**Comment by xmapst** [comment]
> 让我们尽量不要复制不必要的文件。唯一真正的添加应该是golang 1909 docker镜像，甚至可以理想地添加到上游

Normalize each Windows version should have a corresponding, independent file or folder to manage

**Comment by xmapst** [comment]
I think this submission is useful. The adjustment of the directory structure is just to standardize the different Dockerfiles corresponding to each Windows version. Everyone's needs are different, but the structure must be standardized so that most people can easily extract images or package images.

**Pull request closed without merging**